src/pages/projects/user-box/UserBox.module.less

```diff
@@ -0,0 +1,9 @@
+@import '@variables';
+
+.userBoxContainer {
+  flex: 2;
```

Comment:
Why 2? I see no changes without this line

Reply:
default is 1 i think and it's about the width proportions of components on projects page

Reply:
And maybe it wasnt changing anything but now i changed the layout a bit and it matters